grpc/tests/grpctest.cpp

```diff
@@ -46,8 +46,26 @@ class ServiceImpl final : public MyGame::Example::MonsterStorage::Service {
                                   const flatbuffers::BufferRef<Stat> *request,
                                    ::grpc::ServerWriter< flatbuffers::BufferRef<Monster>>* writer)
                                   override {
-    assert(false);  // We're not actually using this RPC.
-    return grpc::Status::CANCELLED;
+       fbb_.Clear();
```

Comment:
Please use similar syntax to the rest of FlatBuffers, i.e. indentation, spacing etc.